pandas/core/strings.py

```diff
@@ -898,23 +898,76 @@ def str_join(arr, sep):
 
 def str_findall(arr, pat, flags=0):
     """
-    Find all occurrences of pattern or regular expression in the
-    Series/Index. Equivalent to :func:`re.findall`.
+    Find all occurrences of pattern or regular expression in the Series/Index.
+
+    Equivalent to apply :func:`re.findall` to all the elements in the
+    Series/Index.
 
     Parameters
     ----------
-    pat : string
-        Pattern or regular expression
-    flags : int, default 0 (no flags)
-        re module flags, e.g. re.IGNORECASE
+    pat : str
+        Pattern or regular expression.
+    flags : int
+        re module flags, e.g. re.IGNORECASE (default is 0, which means
+        no flags).
+    kwargs
+        These parameters will be ignored.
```

Comment:
I know the validation script says this is not documented .. but I am not really sure this is actually useful to put like that in the docs?

Reply:
Tested it quickly, and if you pass another kwarg, it is actually raising an error, so the explanation is also not correct. 

The implementation is a bit complex .. we should try to solve this to don't show the kwargs.